### PR TITLE
Feature/sqs consumer

### PR DESCRIPTION
**Pull request by dylannz**

### CODE REVIEW

File: sqs_consumer/sqs_consumer.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/goamz/goamz/sqs"
+)
+
+// TODO: add tests
```
Comment (dylanmacdonald) [inline sqs_consumer/sqs_consumer.go]: This is a pretty good idea...

File: sqs_consumer/sqs_consumer.go [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+	switch handler := c.handler.(type) {
+	case MessageHandler:
+		if !handler(message) {
+			// Failed to handle message, do nothing. It's the responsibility of the
+			// handler to communicate the failure via logs/bugsnag etc.
+			logger.Debug("failed to handle message")
+			return
+		}
+	case SNSMessageHandler:
+		snsMessage := SnsMessage{}
```
Comment (dylanmacdonald) [inline sqs_consumer/sqs_consumer.go]: Some comments explaining why casting to SNS is necessary, not a blocker though.

File: sqs_consumer/sqs_consumer.go (hunk 1 above)
Comment (dylanmacdonald) [inline sqs_consumer/sqs_consumer.go]: Not a blocker, but definitely something to consider for the future

File: sqs_consumer/sqs_consumer.go (hunk 2 above)
Comment (dylannz) [inline sqs_consumer/sqs_consumer.go]: We're not actually casting anything here. The handler can be one of two types, this is just checking which one it is.

**Pull request merged**